addons/payment/tests/test_payment_token.py

```diff
@@ -31,6 +31,11 @@ def test_cannot_assign_token_to_public_partner(self):
     def test_token_cannot_be_unarchived(self):
         """ Test that unarchiving disabled tokens is forbidden. """
         token = self._create_token(active=False)
+        token.payment_method_id.active = False
```

Comment:
Please extract this case in a separate `test_unarchiving_token_requires_active_payment_method` test to keep investigation and maintenance efforts low.

Reply:
Done the changes :+1: